src/data/mv_polynomial.lean

```diff
@@ -398,6 +398,10 @@ begin
     intro H, rw [_root_.add_eq_zero_iff] at H, exact one_ne_zero H.2 }
 end
 
+lemma exists_coeff_ne_zero (p : mv_polynomial σ α) (h : p ≠ 0) :
```

Comment:
This can be an iff, perhaps more naturally stated in the contrapositive form? `p` should be implicit.

Reply:
I've added the iff versions.